### ISSUE TITLE
Pretraining script fails on Bridges

### ISSUE THREAD

**Issue by iparask** [iceberg-project/Rivers#8]
Hello @samiradaneshgar,

I'm trying to run the pipeline on Bridges. I copied the `2015_07_17` files that exist in the drive and tried to run the pretraining matlab file. Unfortunately, `uigetfile` function does not work without a display. Here is the error from Bridges:
```
Warning: uigetfile is no longer supported when MATLAB is started with the -nodisplay or -noFigureWindows option or there is no display. For more information, see "Changes to
-nodisplay and -noFigureWindows Startup Options" in the MATLAB Release Notes. To view the release note in your system browser, run
web('www.mathworks.com/help/matlab/release-notes.html#br5ktrh-3', '-browser')
> In warnfiguredialog (line 21)
  In uigetfile (line 127)
  In tiling_pretraining (line 8)
  In run (line 91)
Error using javaObjectEDT
Scalar input must be a java object

Error in matlab.ui.internal.dialog.Dialog/getParentFrame (line 46)
               obj.ParentFrame = javaObjectEDT(com.mathworks.hg.peer.utils.DialogUtilities.createParentWindow);

Error in matlab.ui.internal.dialog.FileSystemChooser/getParentFrame (line 129)
                parframe = getParentFrame@matlab.ui.internal.dialog.Dialog(obj);

Error in matlab.ui.internal.dialog.FileChooser/showPeerAndBlockMATLAB (line 113)
            parFrame = getParentFrame(obj);

Error in matlab.ui.internal.dialog.FileChooser/show (line 122)
            obj.showPeerAndBlockMATLAB();

Error in uigetputfile_helper (line 55)
    ufd.show();

Error in uigetfile (line 130)
[filename, pathname, filterindex] = uigetputfile_helper(0, varargin{:});

Error in tiling_pretraining (line 8)
[FileName1, PathName1] = uigetfile('*.tif', 'Select a 16-bit 8-band WV image for training:');

Error in run (line 91)
evalin('caller', strcat(script, ';'));

Exception in thread "AWT-EventQueue-0" java.awt.HeadlessException
	at java.awt.GraphicsEnvironment.checkHeadless(GraphicsEnvironment.java:204)
	at java.awt.Window.<init>(Window.java:536)
	at java.awt.Frame.<init>(Frame.java:420)
	at javax.swing.JFrame.<init>(JFrame.java:233)
	at com.mathworks.mwswing.MJFrame.<init>(MJFrame.java:108)
	at com.mathworks.mwswing.MJFrame.<init>(MJFrame.java:101)
	at com.mathworks.hg.peer.utils.DialogUtilities$1.runWithOutput(DialogUtilities.java:58)
	at com.mathworks.jmi.AWTUtilities$Invoker$2.watchedRun(AWTUtilities.java:475)
	at com.mathworks.jmi.AWTUtilities$WatchedRunnable.run(AWTUtilities.java:436)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
>>
```

I think we should move to either a python function or some use another matlab function that does not require a screen.

**Comment by bspitzbart** [comment]
Is this fixed under PR #10?

**Comment by iparask** [comment]
Yes, it is!
